package.json

```diff
@@ -47,55 +47,55 @@
     "minimist": "^1.2.5",
     "napa": "^3.0.0",
     "popper.js": "^1.16.1",
-    "replace-in-file": "^5.0.2",
-    "stylelint": "^13.3.3",
+    "replace-in-file": "^6.1.0",
+    "stylelint": "^13.6.1",
     "stylelint-config-sass-guidelines": "^7.0.0",
     "stylelint-config-standard": "^20.0.0",
-    "stylelint-order": "^4.0.0",
-    "stylelint-scss": "^3.17.1"
+    "stylelint-order": "^4.1.0",
+    "stylelint-scss": "^3.18.0"
   },
   "dependencies": {
-    "@fortawesome/fontawesome-free": "~5.13.0",
+    "@fortawesome/fontawesome-free": "5.14.0",
     "@ttskch/select2-bootstrap4-theme": "1.3.2",
     "angular": "1.8.0",
-    "angular-sanitize": "1.7.9",
+    "angular-sanitize": "1.8.0",
     "angular-summernote": "0.8.1",
     "backbone": "1.4.0",
-    "bootstrap": "^4.5.0",
-    "bootswatch": "4.4.1",
+    "bootstrap": "4.5.0",
+    "bootswatch": "4.5.0",
     "chart.js": "2.9.3",
     "checklist-model": "1.0.0",
-    "ckeditor4": "4.14.0",
-    "datatables.net": "1.10.20",
-    "datatables.net-bs4": "1.10.20",
+    "ckeditor4": "4.14.1",
+    "datatables.net": "1.10.21",
+    "datatables.net-bs4": "1.10.21",
     "datatables.net-colreorder": "1.5.2",
     "datatables.net-colreorder-dt": "1.5.2",
-    "datatables.net-dt": "1.10.20",
-    "datatables.net-jqui": "1.10.20",
-    "datatables.net-scroller": "2.0.1",
-    "datatables.net-scroller-jqui": "2.0.1",
-    "dropzone": "5.7.0",
+    "datatables.net-dt": "1.10.21",
+    "datatables.net-jqui": "1.10.21",
+    "datatables.net-scroller": "2.0.2",
+    "datatables.net-scroller-jqui": "2.0.2",
+    "dropzone": "5.7.2",
     "dwv": "0.27.0",
-    "flot": "0.8.0-alpha",
+    "flot": "4.2.1",
```

Comment:
this one raises an eyebrow :)
Would test dicom viewer with this; am will to bet it will die badly
(the dicom viewer is finicky in regards to dependencies; @sjpadgett did get dwv to most recent version, so may get lucky I suppose with the non-legacy flot version)

Reply:
tbh there's a lot of dicom dependencies lol 😂